spirv_msl.cpp

```diff
@@ -43,6 +41,20 @@ CompilerMSL::CompilerMSL(vector<uint32_t> spirv_, vector<MSLVertexAttr> *p_vtx_a
 			resource_bindings.push_back(&rb);
 }
 
+CompilerMSL::CompilerMSL(vector<uint32_t> spirv_, vector<MSLVertexAttr> *p_vtx_attrs,
+                         vector<MSLResourceBinding> *p_res_bindings)
+    : CompilerGLSL(move(spirv_))
+{
+	init(p_vtx_attrs, p_res_bindings);
+}
+
+CompilerMSL::CompilerMSL(const uint32_t *ir, size_t word_count, vector<MSLVertexAttr> *p_vtx_attrs,
```

Comment:
Should we make p_vtx_attrs and p_res_bindings vector-less for this variants as well? It doesn't seem like these parameters *could* be raw pointers.